src/deploy.sh

```diff
@@ -804,10 +805,19 @@ function modules_install_to()
     sign_extra_line=2
   fi
 
+  if [[ ${configurations[strip_modules_debug_option]} == 'no' ]]; then
+    strip_modules_debug=''
+    grep --quiet --fixed-strings --line-regexp 'CONFIG_DEBUG_INFO=y' "${PWD}/.config"
+    if [[ "$?" == 0 ]]; then
+      load_module_text "${KW_ETC_DIR}/strings/deploy.txt"
+      warning "${module_text_dictionary[large_initramfs_warning]}"
+    fi
+  fi
+
   if [[ "$local_deploy" == 'local' ]]; then
     cmd='sudo -E make modules_install'
   else
-    cmd="make INSTALL_MOD_PATH=$install_to modules_install"
+    cmd="make ${strip_modules_debug}INSTALL_MOD_PATH=$install_to modules_install"
```

Comment:
Hi, everything looks okay, I just didn't understand why this was not added to the "local" installation.
Could you explain it for me? Is it because of the fore-mentioned "Debian package uses this option by default."?

Reply:
I missed it! Really nice catch! Thanks a lot.